docs/addconsolealias.md

```diff
@@ -56,20 +56,20 @@ BOOL WINAPI AddConsoleAlias(
 The console alias to be mapped to the text specified by *Target*.
 
 *Target* \[in\]  
-The text to be substituted for *Source*. If this parameter is **NULL**, then the console alias is removed.
+The text to be substituted for *Source*. If this parameter is `NULL`, then the console alias is removed.
 
 *ExeName* \[in\]  
 The name of the executable file for which the console alias is to be defined.
 
 ## Return value
 
-If the function succeeds, the return value is **TRUE**.
+If the function succeeds, the return value is `TRUE`.
 
-If the function fails, the return value is **FALSE**. To get extended error information, call [**GetLastError**](https://msdn.microsoft.com/library/windows/desktop/ms679360).
+If the function fails, the return value is `FALSE`. To get extended error information, call [``GetLastError`](https://msdn.microsoft.com/library/windows/desktop/ms679360).
 
 ## Remarks
 
-To compile an application that uses this function, define **\_WIN32\_WINNT** as 0x0501 or later. For more information, see [Using the Windows Headers](https://msdn.microsoft.com/library/windows/desktop/aa383745).
+To compile an application that uses this function, define `\_WIN32\_WINNT` as 0x0501 or later. For more information, see [Using the Windows Headers](https://msdn.microsoft.com/library/windows/desktop/aa383745).
```

Comment:
if we make this change, do we need to remove `\` from the escaped underscores? they might come out like `\_`

Reply:
Hmm.. what's the proper definition?
\_WIN32\_WINNT or \WIN32\WINNT?

Reply:
with `_`s :smile:

Reply:
Sorry -- with ONLY underscores. `_WIN32_WINNT`!

Reply:
Ahh-- yes, I'll need to go through and remove the escaped underscores. Great catch! Thank you!!